source/reference/minio-mc/mc-support-logs-enable.rst

```diff
@@ -0,0 +1,61 @@
+==========================
+``mc support logs enable``
+==========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support logs enable
+
+Description
+-----------
+
+Use the :mc-cmd:`mc support logs enable` command to allow real-time MinIO logs to upload to |subnet|.
+
+.. include:: /includes/common-mc-support.rst
+   :start-after: start-support-logs-opt-in
+   :end-before: end-support-logs-opt-in
+   
+.. include:: /includes/common-mc-support.rst
+   :start-after: start-minio-only
+   :end-before: end-minio-only
+
+.. note:: 
+
+   By default, MinIO does not scrub the logs uploaded to SUBNET.
+
+   To hide sensitive information in the logs, start the server with the :mc-cmd:`~mc server --anonymous` flag.
+
+Example
+-------
+
+Upload Real-Time Logs to SUBNET
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following command starts sending console logs to SUBNET for the alias ``minio1``.
```

Comment:
/s/console/MinIO Server

Since the Console is a separate binary. 'console' here probably means the STDOUT of the server, but i'll bet users will get confusedc.

Reply:
Yep, I'd bet you'd be right.